client/src/components/Alert/Alert.tsx

```diff
@@ -11,10 +11,12 @@ const Alert: React.FunctionComponent<AlertProps> = ({
   clearAlert,
   message,
   color
-}) => (
-  <div>
-    {message && <UncontrolledAlert color={color}>{message}</UncontrolledAlert>}
-  </div>
-);
+}) => {
+  return (
+    <div>
```

Comment:
do we need this div?
can we do like this

`=> {message && <UncontrolledAlert .. >}

Reply:
I removed return statement, but left div, cause without it I had an error.